nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -383,10 +386,8 @@ def _assign_mixed_precision(all_weight_params: List[WeightNodeParams], ratio: fl
             current_ratio = (num_weights_in_4bit + weight_param.num_weights) / num_internal_weights
             if current_ratio >= ratio:
                 break
-            weight_param.compression_config = nf4_config
+            weight_param.compression_config = primary_config
             num_weights_in_4bit += weight_param.num_weights
-
     else:
```

Comment:
A matter of style:
Do you considered to move this short else body forward and to use `return` instead of else body?

Reply:
ok, saved 1 tab